docs/source/operations/orchestrator/deployment.rst

```diff
@@ -0,0 +1,223 @@
+
+      .. code-block:: ini
+
+         [ req ]
+         default_bits		= 2048
+         default_md		= sha256
+         distinguished_name	= req_distinguished_name
+
+         [ req_distinguished_name ]
+
+         [ v3_ca ]
+         basicConstraints = critical,CA:TRUE
+         subjectKeyIdentifier = hash
+         authorityKeyIdentifier = keyid:always,issuer:always
+         keyUsage = cRLSign, keyCertSign
+
+   #. Generate a private key for signing certificates:
+
+      .. code-block:: bash
+
+         openssl genrsa -out orchestrator-ca.key 2048
+
+   #. Generate your Certificate Authority certificate:
+        
+      .. code-block:: bash
+
+        openssl req -new -x509 -days 365 -sha256 -key orchestrator-ca.key -extensions v3_ca -config example-openssl.cnf -subj "/C=FR/ST=Loire-Atlantique/L=Nantes/O=ORC-ORG-NAME/CN=Orchestrator Root CA" -out orchestrator-ca.crt
+
+      | Replace ``ORC-ORG-NAME`` with the name of your organization.
+
+#. Generate a certificate for the Orchestrator
+
+   #. Generate a certificate signing request:
+
+      .. code-block:: bash
+
+         openssl req -newkey rsa:2048 -nodes -keyout orchestrator-tls.key -subj "/C=FR/ST=Loire-Atlantique/L=Nantes/O=ORC-ORG-NAME/CN=HOSTNAME" -out orchestrator-cert.csr
+
+      | Replace ``ORC-ORG-NAME`` with the name of your organization.
+      | Replace ``HOSTNAME`` with the hostname of your Orchestrator as in the ingress configuration.
+      
+      This will generate a private key for the orchestrator and a certificate signing request.
+      You should have two new files in your current directory ``orchestrator-tls.key`` and ``orchestrator-cert.csr``.
+
+   #. Sign the request with the Certificate Authority key:
+
+      .. code-block:: bash
+
+         openssl x509 -req -days 365 -in orchestrator-cert.csr -CA orchestrator-ca.crt -CAkey orchestrator-ca.key -CAcreateserial -out orchestrator-tls.crt -extfile <(printf "subjectAltName=DNS:HOSTNAME")
+
+      | Replace ``HOSTNAME`` with the hostname of your Orchestrator.
+
+   #. Delete the Certificate Signing Request:
+
+      .. code-block:: bash
+
+         rm orchestrator-cert.csr orchestrator-ca.srl
+
+#. Create a Kubernetes ConfigMap for the CA certificate:
+   
+   .. code-block:: bash
+      
+      kubectl create configmap orchestrator-tls-cacert --from-file=orchestrator-ca.crt
+
+#. Create a Kubernetes Secret for the orchestrator TLS key and certificate:
+
+   .. code-block:: bash
+      
+      kubectl create secret tls orchestrator-tls-server-pair --cert=orchestrator-tls.crt --key=orchestrator-tls.key
+
+
+
+
+We want to validate clients identity usigng mTLS so we need to enable some additional settings:
+In the same values file add:
+
+```
+orchestrator:
+  verifyClientMSPID: true
+```
+
+We enable TLS for the server:
+in te same values.yaml file
+```
+orchestrator:
+  verifyClientMSPID true
+  tls:
+    enabled: true
+```
+
+Now for this to work we need to generate some certificates.
```

Comment:
Can we specify "come certificates"? What for?

Reply:
This is part of the non-formatted doc.